python/ray/tests/kuberay/test_autoscaling_config.py

```diff
@@ -35,6 +35,20 @@ def get_basic_ray_cr() -> dict:
     )
     gpu_group["maxReplicas"] = 200
     config["spec"]["workerGroupSpecs"].append(gpu_group)
+    tpu_group = copy.deepcopy(config["spec"]["workerGroupSpecs"][0])
```

Comment:
Do we need to set `numOfHosts`?

Reply:
Done in [08b8eb9](https://github.com/ray-project/ray/pull/43470/commits/08b8eb9016a87d1751461adef7cfd1ae65304494).